pip_audit/_cli.py

```diff
@@ -401,6 +410,10 @@ def audit() -> None:
             else:
                 spec = cast(ResolvedDependency, spec)
                 state.update_state(f"Auditing {spec.name} ({spec.version})")
+            if args.ignore_vuln:
+                filtered_vulns = [v for v in vulns if v.id not in args.ignore_vuln]
```

Comment:
Nit: We also probably need to check the each result's `aliases` set here, since the ID might be a non-PYSEC ID (or the vuln might not have a PYSEC ID at all).

That's a bit messy to do inline, so we might want to consider a new `VulnerabilityResult.has_id` API.

Reply:
Ok, so if I understand correctly this `has_id` (or `has_ids`) method should take a list of IDs (PYSEC, CVE, etc.) and check whether the input list intersects with the `VulnerabilityResult`'s id + aliases? and if so return `True`, `False` otherwise?

Reply:
Yep, correct. I guess we need to check the intersection for all ignored IDs, so maybe `has_any_id(set)`?

Reply:
I added this as a separate commit, let me know if you wish to have it squashed